bin/generate-readme

```diff
@@ -38,13 +38,13 @@ insert_version_urls() {
     log(pkg.devDependencies);
   ')"
 
-  local -r pattern1='^<h4 name="([^"]*)#([^"]*)">'
+  local -r pattern1='^(#### <a name=")([^"]*)(".*)$'
   local -r pattern2='^(.+) ([Vv]):([^/]+)/([^# ]+)'  # V => 1.2.3 ; v => v1.2.3
   local -r pattern3='^[0-9]+[.][0-9]+[.][0-9]+$'
   while IFS= read -r line ; do
     if [[ "$line" =~ $pattern1 ]] ; then
       local matches=("${BASH_REMATCH[@]}")
-      echo "<h4 name="'"'"${matches[1]}.prototype.${matches[2]}"'"'">${line:${#matches[0]}}"
+      echo "${matches[1]}${matches[2]/'#'/.prototype.}${matches[3]}"
```

Comment:
This is a simpler approach: rather than have the pattern match only those names which contain `#`, we match all names and perform the substitution unconditionally.

For those unfamiliar with the `${.../.../...}` syntax, it replaces the first occurrence of a substring with a replacement string. For example:

```console
$ echo "${SHELL/a/X}"
/usr/locXl/bin/bash
```